space/space.go

```diff
@@ -81,19 +81,20 @@ func Createspace(db *sql.DB, space structs.Spacespec, berr binding.Errors, r ren
 		utils.ReportError(err, r)
 		return
 	}
-	if os.Getenv("IMAGE_PULL_SECRET") != "" {
-		// This secret created used to be passed in to AddImagePullSecretToSpace, but that method
-		// did nothing with it, so we don't in the refactor.
-		if err := rt.CopySecret(os.Getenv("IMAGE_PULL_SECRET"), "akkeris-system", space.Name); err != nil {
+
+	ipss := []structs.Namespec{}
```

Comment:
What does `ipss` stand for here? Is that something everyone else would be familiar with? If so, great. If not, maybe consider changing the variable name to something more descriptive? Just a suggestion, so your call.

Reply:
ImagePullSecretS - plural

Reply:
I should use more descriptive names.